src/stories/containers/Finances/components/DelegateExpenseTrend/HeaderDelegateExpense.stories.tsx

```diff
@@ -0,0 +1,249 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import lightTheme from '@ses/styles/theme/light';
+import HeaderDelegateExpense from './HeaderDelegateExpense';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/NewFinances/HeaderDelegateExpense',
+  component: HeaderDelegateExpense,
+  parameters: {
+    chromatic: {
+      viewports: [1194, 1280, 1440, 1920],
+    },
+  },
+} as ComponentMeta<typeof HeaderDelegateExpense>;
+
+const variantsArgs = [
+  {
+    columns: [
+      {
+        header: 'Contributors',
+        styles: {
+          boxSizing: 'border-box',
+          minWidth: 228,
+          paddingLeft: 16,
+        },
+        sortReverse: true,
+      },
+      {
+        header: 'Reporting Month',
+        styles: {
+          width: 170,
+          marginLeft: 112,
+
+          [lightTheme.breakpoints.up('desktop_1280')]: {
+            marginLeft: 122,
+          },
+          [lightTheme.breakpoints.up('desktop_1440')]: {
+            marginLeft: 126,
+          },
+        },
+        sortReverse: true,
+      },
+      {
+        header: 'Total Actuals',
+
+        styles: {
+          width: 170,
+          marginLeft: -18,
+
+          [lightTheme.breakpoints.up('desktop_1280')]: {
+            marginLeft: -4,
+          },
+          [lightTheme.breakpoints.up('desktop_1440')]: {
+            marginLeft: 12,
+            justifyContent: 'center',
+          },
+        },
+        sortReverse: true,
+      },
+      {
+        header: 'Status',
+
+        styles: {
+          width: 173,
+          marginLeft: -6,
+
+          [lightTheme.breakpoints.up('desktop_1280')]: {
+            marginLeft: 2,
+          },
+          [lightTheme.breakpoints.up('desktop_1440')]: {
+            marginLeft: 12,
+            justifyContent: 'center',
+          },
+        },
+        sortReverse: true,
+      },
+      {
+        header: 'Last Modified',
+
+        styles: {
+          width: 173,
+          marginLeft: 10,
+
+          [lightTheme.breakpoints.up('desktop_1280')]: {
+            marginLeft: 22,
+          },
+          [lightTheme.breakpoints.up('desktop_1440')]: {
+            marginLeft: 92,
+            justifyContent: 'center',
+          },
+        },
+        sortReverse: true,
+      },
+    ],
```

Comment:
would be good to export this in the component and use it here, if something of this changes here chromatic will pass but the actual component is going to have an issue that is not going to be detected, and if it is changed in the component but not here, then we're going to have an issue here at chromatic

Reply:
done